packages/lab/src/typography/Text.css

```diff
@@ -103,35 +103,42 @@ h4.uitkText small,
 /* Label-Caption */
 label.uitkText,
 .uitkText-label {
+  --uitkText-line-height: var(--uitk-text-caption-lineHeight);
   font-size: var(--uitk-text-caption-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-caption-lineHeight));
 }
 
 /* Help text */
 .uitkText-helpText.uitkText {
-  font-style: var(--uitk-text-help-fontStyle);
+  --uitkText-line-height: var(--uitk-text-help-lineHeight);
   color: var(--uitk-text-secondary-foreground);
+  font-style: var(--uitk-text-help-fontStyle);
   font-size: var(--uitk-text-help-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-help-lineHeight));
 }
 
 /* Figure 1 */
 .uitkText-figure1.uitkText {
+  --uitkText-line-height: var(--uitk-text-figure1-lineHeight);
   font-size: var(--uitk-text-figure1-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure1-lineHeight));
   font-weight: var(--uitk-text-figure1-fontWeight);
 }
 
 /* Figure 2 */
 .uitkText-figure2.uitkText {
+  --uitkText-line-height: var(--uitk-text-figure2-lineHeight);
   font-size: var(--uitk-text-figure2-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure2-lineHeight));
   font-weight: var(--uitk-text-figure2-fontWeight);
 }
 
 /* Figure 3 */
 .uitkText-figure3.uitkText {
+  --uitkText-line-height: var(--uitk-text-figure3-lineHeight);
   font-size: var(--uitk-text-figure3-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure3-lineHeight));
   font-weight: var(--uitk-text-figure3-fontWeight);
 }
+
+.uitkText {
+  line-height: var(--uitkText-line-height);
+}
+.backwardsCompat .uitkText {
```

Comment:
should have these together .backwardsCompat.uitkText

Reply:
.uitkText may be included inside main components, so we need to make sure the rule still applies

Reply:
in that case it should be the component thats unique text is backwards compatible (for example - not a true case that ive checked but there will be similar scenarios - h1 text is the same everywhere as we have now defined, but maybe in a dialog title someone applied backwards compatibility, and that's where the inner h1 text needs adjusting, not in the text component itself, so in .backwardsCompat.uitkDialog-title the --uitkText api is used)

Reply:
Normally we would apply the backwardsCompat class on the Dialog, not on DialogTitle. Even so, the h1 would be a child of DialogTitle. There is no .backwardsCompat.uitkText as there is no uitkText in TK1.
I understand your logic, but I think this case is a little different from the pattern we have on other components.
Maybe we should have a call and discuss this in more details

Reply:
that makes sense.. (in that case, do we call it 'backwards compatible'?) 
i agree!

Reply:
I've added `.backwardsCompat .uitkText` to make sure all the components which now include .uitkText will be backwards compatible with the TK1 version. I'm open to other suggestions :)

Reply:
sounds good, let's make the API token --uitkText-lineHeight, without -backwardsCompat, and then people can set that token within backwardsCompat class - this is the convention we have taken in other places to avoid bloating css with too many different variables